### ISSUE TITLE
Unexpected extra space in line-up indentation

### ISSUE THREAD

**Issue by oberpar** [perltidy/perltidy#106]
Using line-up indentation (either `-lp` or `-xlp`) on a parameter list spanning exactly two lines causes `perltidy` (v20220613.03) to generate one extra column of indentation on the second line.

### Example

Command-line:

```
$ perltidy -xlp -pt=2
```

Expected result:

```perl

A("0123456789012345678901234567890123456789",
  "0123456789012345678901234567890123456789");
```

Actual result:

```perl
A("0123456789012345678901234567890123456789",
   "0123456789012345678901234567890123456789");
```

### Additional observations

1. For an indentation level of `-i=n`, the extra column only occurs for resulting line-up indentation levels between 2 and n-1

   Command-line:

   ```
   $ perltidy -i=8 -xlp -pt=2
   ```

   Result:

   ```perl
   # Expect line-up indentation 7, got 8
   AAAAAA("0123456789012345678901234567890123456789",
           "0123456789012345678901234567890123456789");

   # Expect line-up indentation 8, got 8
   AAAAAAA("0123456789012345678901234567890123456789",
           "0123456789012345678901234567890123456789");
   ```

2. Unexpected indentation does not occur if there are more than 2 lines in the list

   Command-line:

   ```
   $ perltidy -xlp -pt=2
   ```

   Result:

   ```perl
   A(
      "0123456789012345678901234567890123456789",
      "0123456789012345678901234567890123456789",
      "0123456789012345678901234567890123456789"
   );
   ```

3. A similar type of unexpected indentation also occurs for concatenations spanning multiple lines if the concatenation operator is found at the end of a line (`-wba=.`). For concatentation, the problem remains even with more than 2 lines.

   Command-line:

   ```
   $ perltidy -xlp -pt=2 -wba=.
   ```

   Result:

   ```perl
   A("0123456789012345678901234567890123456789" .
      "0123456789012345678901234567890123456789" .
      "0123456789012345678901234567890123456789");
   ```

**Comment by shancock9** [comment]
Thanks, I'll take a look at it.

**Comment by shancock9** [comment]
There is a patch to fix this in tagged version 20220613.04,

https://github.com/perltidy/perltidy/releases/tag/20220613.04

**Comment by shancock9** [comment]
Version 20221112 has this update.
